dev_guide/application_health.adoc

```diff
@@ -44,6 +44,21 @@ Set a readiness check by configuring the
 
 |===
 
+The exact timing of a probe is controlled by two fields,
+both expressed in units of seconds:
+
+|====
+| Field | Meaning
+
+| `initialDelaySeconds`
+| How long to wait after the container starts to begin the probe.
+
+| `timeoutSeconds`
```

Comment:
[Markup for fields/parameters](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference):

```
`*field_name*`
```